### PR TITLE
Fixing a warning that says unexpected type when creating an actionrow.

### PR DESCRIPTION
**Pull request by Ashen-M**
Changed the ComponentHolder class to be `typing.Union[BaseComponent, ActionRow]` from `typing.List[BaseComponent]`. Easy kludge fix.

### CODE REVIEW

File: voxelbotutils/cogs/utils/interactions/components/action_row.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def to_dict(self):
         }
 
     @classmethod
-    def from_dict(cls, data: dict):
+    def from_dict(cls, data: list):
```
Comment (4Kaylum) [inline voxelbotutils/cogs/utils/interactions/components/action_row.py:29]: Is this accurate? An ActionRow is itself an object with a components attr.

File: voxelbotutils/cogs/utils/interactions/components/models.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import re
 import typing
 import json
+from voxelbotutils.cogs.utils.interactions.components.action_row import ActionRow
```
Comment (4Kaylum) [inline voxelbotutils/cogs/utils/interactions/components/models.py:4]: This should be a relative import.

File: voxelbotutils/cogs/utils/interactions/components/action_row.py [hunk 3]
```diff
@@ -49,7 +49,7 @@ def to_dict(self):
         return [i.to_dict() for i in self.components]
 
     @classmethod
-    def from_dict(cls, data: list):
+    def from_dict(cls, data: dict):
```
Comment (4Kaylum) [inline voxelbotutils/cogs/utils/interactions/components/action_row.py:52]: Again, is this accurate? MessageComponents is a list of ActionRow objects at best.

File: voxelbotutils/cogs/utils/interactions/components/models.py [hunk 4]
```diff
@@ -81,7 +82,7 @@ class ComponentHolder(BaseComponent):
     A message component that holds other message components.
     """
 
-    def __init__(self, *components: typing.List[BaseComponent]):
+    def __init__(self, *components: typing.Union[BaseComponent, ActionRow]):
```
Comment (4Kaylum) [inline voxelbotutils/cogs/utils/interactions/components/models.py:85]: ActionRow inherits from BaseComponents - this should just be `*components: BaseComponent` so as to futureproof for other layout components.

**Review by 4Kaylum — CHANGES_REQUESTED** [review]

File: voxelbotutils/cogs/utils/interactions/components/action_row.py (hunk 1 above)
Comment (Ashen-M) [reply voxelbotutils/cogs/utils/interactions/components/action_row.py:29]: Didn't mean to change that line.

File: voxelbotutils/cogs/utils/interactions/components/action_row.py (hunk 3 above)
Comment (Ashen-M) [reply voxelbotutils/cogs/utils/interactions/components/action_row.py:52]: It's returning it to as-written in the library already: [per here](https://github.com/Voxel-Fox-Ltd/VoxelBotUtils/blob/master/voxelbotutils/cogs/utils/interactions/components/action_row.py#L29)

File: voxelbotutils/cogs/utils/interactions/components/models.py (hunk 4 above)
Comment (Ashen-M) [reply voxelbotutils/cogs/utils/interactions/components/models.py:85]: That's true, updated.

File: voxelbotutils/cogs/utils/interactions/components/models.py (hunk 2 above)
Comment (Ashen-M) [reply voxelbotutils/cogs/utils/interactions/components/models.py:4]: Removed per changes made [here](https://github.com/Voxel-Fox-Ltd/VoxelBotUtils/pull/112#discussion_r671605650).

**Pull request closed without merging**